api/Element.json

```diff
@@ -5929,6 +5929,41 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "escape_lt_gt_in_attributes": {
+          "__compat": {
+            "description": "Serializes `<` and `>` in attributes as `&lt;` and `&gt;`.",
+            "spec_url": "https://github.com/whatwg/html/issues/6235",
+            "support": {
+              "chrome": {
+                "version_added": false
```

Comment:
This is enabled behind the pref `#enable-experimental-web-platform-features` in https://chromium-review.googlesource.com/c/chromium/src/+/4362958 (for all these additions). But I don't know how we find the version - advice?

Reply:
According to https://chromiumdash.appspot.com/commit/1ac10dac9451ebe4296d50fc05f18e0d4e17197b, this (unshipped) feature landed in Chrome 114.

Reply:
Done. Thank you.